spec/openapi.yaml

```diff
@@ -222,37 +119,31 @@ paths:
                 type: object
                 properties:
                   temperature:
-                    $ref: "#/components/schemas/OctoTemperatureState"
+                    $ref: "#/components/schemas/TemperatureState"
                   sd:
-                    $ref: "#/components/schemas/OctoSDState"
+                    $ref: "#/components/schemas/SDState"
                   state:
-                    $ref: "#/components/schemas/OctoPrinterState"
+                    $ref: "#/components/schemas/PrinterState"
         403:
           $ref: "#/components/responses/Unauthorized"
+
   /api/printer/sd:
     get:
-      tags:
-        - octoprint
       summary: Retrieves the current state of the printer’s SD card.
-      security:
-        - ApiKeyAuth: []
       responses:
         200:
           description: OK
           content:
             application/json:
               schema:
                 type: object
-                $ref: "#/components/schemas/OctoSDState"
+                $ref: "#/components/schemas/SDState"
         403:
           $ref: "#/components/responses/Unauthorized"
-  /api/job:
+
+  /api/job:   # this should be used instead of /files/preview to determine if project was selected. If "file" object is not null, project was previously selected
```

Comment:
Is this for use instead of /api/progress?

Reply:
How I will get the preview image?

Reply:
> Is this for use instead of /api/progress?
Yes

and 

> How I will get the preview image?
https://github.com/prusa3d/Prusa-Connect-Local/pull/45/files#diff-dfce4d9d11a31be72739d767812e1b3435a06045c19be411a28b517d2c208001R840